CHANGELOG.md

```diff
@@ -12,9 +12,18 @@ While still incomplete this lays the foundation for running druid on Linux witho
 
 #### Mostly complete Wasm backend for druid-shell. ([#759])
 
-[@elrnv] continued the work of [@tedsta] and implemented a mostly complete Wasm backend and enabled all druid examples to [run in the browser](https://elrnv.github.io/druid-wasm-examples/).
+[@elrnv] continued the work of [@tedsta] and implemented a mostly complete Wasm backend and enabled
+all druid examples to [run in the browser](https://elrnv.github.io/druid-wasm-examples/).
 
-While some features like the clipboard, menus or file dialogs are not yet available, all fundamental features are there.
+While some features like the clipboard, menus or file dialogs are not yet available,
+all fundamental features are there.
+
+#### Using Core Graphics on macOS. ([#905])
+
+[@cmyr] continued the work of [@jrmuizel] and implemented Core Graphics support for piet in
+[piet#176](https://github.com/linebender/piet/pull/176).
```

Comment:
Maybe make it more prominent that this means that druid now actually uses Core Graphics and no longer needs Cairo on mac? Because that seems to be the most awesome part of it.

Reply:
I added an additional note about it.